issues: tree view fixes re: #679

the-j0k3r opened the pull request:
Test these out @tambry 

gif shows initially zoom at 100% and each full scroll into oposite direction increases 10% zoom up to 130% zoom with no gaps.

Browser FF 61.0
 
zoom 100% initial scroll to bottom
zoom 110% scroll to top
zoom 120% scroll to bottom
zoom 130% scroll to top

the gif is 15MB (squeezed from original 37MB) so download it...

https://www.dropbox.com/s/5pypt47ab9albi6/treeview.gif?dl=1

tambry commented:
I don't think it improves anything. In fact the gap between the dotted pictures increases and becomes more obvious. I think my original fix, but with the now pre-inverted backgrounds would be better. I'll make a PR for that.

Before:
![2018-07-07_13-01-25_chrome](https://user-images.githubusercontent.com/1827708/42409641-13b77a40-81e6-11e8-9cb3-a749d1255154.png)

After:
![2018-07-07_13-02-10_chrome](https://user-images.githubusercontent.com/1827708/42409642-16848ace-81e6-11e8-8806-cc45b7571516.png)

the-j0k3r commented:
> I think my original fix, but with the now pre-inverted backgrounds would be better. I'll make a PR for that.

that results in broken background in Firefox as well.

Mottie commented:
What if we reduced the height of the image? Or set the background size to cover?

the-j0k3r commented:
> What if we reduced the height of the image

I reduced the heigh of the image here physically to 25px from 26px removing the gap at top edge between canvas and dot, so perhaps reduce it also in CSS? let me push that in here.

the-j0k3r commented:
`background-size: cover !important;` screws with the center alignment and any more editing to files itself will be a very deep rabbit hole/can of worms (from experience with this issue in other projects), so I will bow out of that scenario.

Open to other suggestions though.

the-j0k3r commented:
`	background-position: -2px center !important;` is interesting.

![popover](https://user-images.githubusercontent.com/31389848/42412680-e39100e8-8208-11e8-8bb1-2944e49058e0.gif)

@tambry I reverted gifs to latest default and added tweak above, I wonder if that has a positive effect on the issue you see despite it just aligning the image  horizontally.

the-j0k3r commented:
Im out of ideas ATM so any suggestions welcome.

the-j0k3r commented:
> Looks good enough to me.

so I pushed width changes in, any other changes pls make some suggestion re: gaps I cant see any in FF

Mottie commented:
Are we all happy with these tweaks? I'm a bit under the weather, so I'll let you guys/gals argue about the minor details.

the-j0k3r commented:
>Are we all happy with these tweaks? I'm a bit under the weather, so I'll let you guys/gals argue about the minor details.

rahrahrah, argue argue argue, Im foine. here

the-j0k3r commented:
Thanks @tambry
